client_app/src/components/authentication/UnifiedDashboard.js

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { SERVER } from "../../config";
+import { Container, Button, Row, Col, Card } from "react-bootstrap";
+
+const UnifiedDashboard = () => {
+  const [roles, setRoles] = useState([]);
+  const [shelterIds, setShelterIds] = useState([]);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const fetchPermissions = async () => {
+      const token = localStorage.getItem("token");
+      try {
+        const response = await fetch(`${SERVER}/user_permission`, {
+          headers: {
+            Authorization: `Bearer ${JSON.parse(token)}`,
```

Comment:
I'm pretty sure this is incorrect. In other places within the client app, headers are retrieved as: `getAuthHeader();` (this function is defined in `client_app/authentication/getAuthHeader.js`